docs/examples/add-to-your-application/go/go.sum

```diff
@@ -0,0 +1,94 @@
+github.com/andybalholm/brotli v1.0.2/go.mod h1:loMXtMfwqflxFJPmdbJO0a3KNoPuLBgiu3qAvBg8x/Y=
```

Comment:
This file seems to be gitignored in the quickstart -- do we want it here?

Reply:
Nope will remove